### PR TITLE
Nimanita patch 1 : Added general-motors.md file

### PR DESCRIPTION
**Pull request by Nimanita**

### CODE REVIEW

**Comment by Nimanita** [comment]
@kelseyhuse30  Please add `hacktoberfest-accepted` label to it also.
Thank you

File: companies/costco.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
```
Comment (kelseyhuse30) [inline companies/costco.md:1]: I'm not sure how the Costco file got stuck in this PR as well :( @Nimanita do you think you could delete the file from this branch?

**Comment by Nimanita** [comment]
@kelseyhuse30  I have deleted costco.md file from this branch.

File: companies/general-motors.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+fortune_rank: 18
+name: General Motors
+slug: general motors
```
Comment (kelseyhuse30) [inline companies/general-motors.md:4]:
```suggestion
slug: general-motors

```

**Review by kelseyhuse30 — CHANGES_REQUESTED** [review]

File: companies/general-motors.md (hunk 2 above)
Comment (kelseyhuse30) [reply companies/general-motors.md:4]: @Nimanita One tiny update and then I will merge :) Would you like to commit the suggestion here or push an update for the slug?

Comment (Nimanita) [reply companies/general-motors.md:4]: I will commit the suggestion.It will be easy for me.

**Review by kelseyhuse30 — APPROVED** [review]

**Pull request merged**